internal/services/availability_status_service.go

```diff
@@ -24,6 +23,5 @@ func AvailabilityStatus(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	background.EnqueueAvailabilityStatusRequest(&msg)
-	metrics.IncTotalReceivedAvailabilityCheckReqs(err)
 	writeOk(w, r)
 }
```

Comment:
Are we giving up on this counter? Why?

Reply:
I mean we could still count all kafka messages received to get what we want.

In addition, we need to add more metrics in order to count success/failure rate without these incorrect source IDs, as Ondra mentioned.

Reply:
this metric is identical to this one and thus redundant: 
provisioning_http_request_total{exported_service="provisioning", path="/api/provisioning/v1/availability_status/sources/"}

Reply:
Oh right, I missed this one.